packages/device_info_plus/device_info_plus/example/integration_test/device_info_plus_test.dart

```diff
@@ -65,4 +66,121 @@ void main() {
       }
     }
   });
+
+  testWidgets('iOS', (WidgetTester tester) async {
+    if (Platform.isIOS) {
+      expect(iosInfo.name, isNotNull);
+      expect(iosInfo.systemName, equals('iOS'));
+      expect(iosInfo.systemVersion, isNotNull);
+      expect(iosInfo.model, equals('iPhone'));
+      expect(iosInfo.localizedModel, equals('iPhone'));
+
+      expect(iosInfo.identifierForVendor, isNotNull);
+      expect(iosInfo.isPhysicalDevice, isNotNull);
+
+      expect(iosInfo.utsname.sysname, equals('Darwin'));
+      expect(iosInfo.utsname.nodename, isNotNull);
+      expect(iosInfo.utsname.release, isNotNull);
+      expect(iosInfo.utsname.version, isNotNull);
+      expect(iosInfo.utsname.machine, isNotNull);
+    }
+  });
+
+  testWidgets('android', (WidgetTester tester) async {
```

Comment:
Can you change the test name to something else? For example: "check android info values are set".

Reply:
Got~